test/Microsoft.Process.Environment.Tests/EnvironmentManagerCentennialTests.cpp

```diff
@@ -0,0 +1,26 @@
+﻿#include <pch.h>
+#include <EnvironmentManagerCentennialTests.h>
+
+using namespace winrt::Microsoft::ProjectReunion;
+
+namespace ProjectReunionCppTest
+{
+    void EnvironmentManagerCentennialTests::CentennialTestGetForProcess()
+    {
+        EnvironmentManager environmentManager = EnvironmentManager::GetForProcess();
```

Comment:
NIT: `EnvironmentManager environmentManager{ EnvironmentManager::GetForProcess() };`

See https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html#es23-prefer-the--initializer-syntax

Same comment applies throughout